### PR TITLE
Front&backend

### PR DESCRIPTION
**Pull request by jiangjie1111**

### CODE REVIEW

File: .DS_Store
Comment (Routhleck) [inline .DS_Store:1]: This file is unnecessary

**Review by heyi755 — APPROVED** [review]
all ok

File: backend/.DS_Store
Comment (Routhleck) [inline backend/.DS_Store:1]: This file is unnecessary

**Review by Routhleck — CHANGES_REQUESTED** [review]
Just delete two .DS_Store file is ok

**Pull request merged**